js/stores/eventStore.js

```diff
@@ -98,8 +98,10 @@ const doAction = (action) => {
       eventState = eventState.set('page_load', eventState.get('page_load').push(pageLoadEvent))
       break
     case WindowConstants.WINDOW_SET_FOCUSED_FRAME:
-      addPageView(action.frameProps.get('location'))
-      lastActiveTabId = action.frameProps.get('tabId')
+      if (action.frameProps) {
```

Comment:
since i am not familiar with the code, and you wrote it, let me defer the changes to you... less likely to have an inadvertent mishap... thanks!